modules/minutemediaBidAdapter.js

```diff
@@ -428,5 +428,9 @@ function generateGeneralParams(generalObject, bidderRequest) {
     generalParams.page_url = deepAccess(bidderRequest, 'refererInfo.page') || window.location.href
   }
 
+  if (config.getConfig('coppa') === true) {
```

Comment:
this could actually change on requestBids, please read off bidrequest.ortb2.regs